libr/util/print.c

```diff
@@ -1581,9 +1582,57 @@ R_API void r_print_progressbar(RPrint *p, int pc, int _cols) {
 	p->cb_printf ("]");
 }
 
+/* TODO: handle screen width */
```

Comment:
Use `hex.cols` eval var for this. use p->coreb.cfgGet(p->coreb->core, "hex.cols)` instead of hardcode the 78 in there. Bear in mind that hex.cols is 16 by default because its used for printing hexdumps. but as long as hexdumps are a common visual metric that variable is used to determine the width of other printed stuff.